packages/protoc-gen-es/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "files": ["src/protoc-gen-es-plugin.ts"],
+  "files": ["src/protoc-gen-es-plugin.ts", "src/transpile.ts"],
```

Comment:
Had to add this temporarily so that linting passes.  Once we decide where to put the `transpile.ts` file, we can fix this.